content/en/docs/reference/command-line-tools-reference/feature-gates/index.md

```diff
@@ -102,171 +102,9 @@ A *General Availability* (GA) feature is also referred to as a *stable* feature.
-- {{< feature-gate-description name="RemoveSelfLink" >}}
-- {{< feature-gate-description name="RotateKubeletServerCertificate" >}}
-- {{< feature-gate-description name="RuntimeClassInImageCriApi" >}}
-- {{< feature-gate-description name="SchedulerQueueingHints" >}}
-- {{< feature-gate-description name="SecurityContextDeny" >}}
-- {{< feature-gate-description name="SELinuxMountReadWriteOncePod" >}}
-- {{< feature-gate-description name="SeparateTaintEvictionController" >}}
-- {{< feature-gate-description name="ServerSideApply" >}}
-- {{< feature-gate-description name="ServerSideFieldValidation" >}}
-- {{< feature-gate-description name="ServiceAccountTokenJTI" >}}
-- {{< feature-gate-description name="ServiceAccountTokenNodeBinding" >}}
-- {{< feature-gate-description name="ServiceAccountTokenNodeBindingValidation" >}}
-- {{< feature-gate-description name="ServiceAccountTokenPodNodeInfo" >}}
-- {{< feature-gate-description name="ServiceNodePortStaticSubrange" >}}
-- {{< feature-gate-description name="SidecarContainers" >}}
-- {{< feature-gate-description name="SizeMemoryBackedVolumes" >}}
-- {{< feature-gate-description name="SkipReadOnlyValidationGCE" >}}
-- {{< feature-gate-description name="StableLoadBalancerNodeSet" >}}
-- {{< feature-gate-description name="StatefulSetAutoDeletePVC" >}}
-- {{< feature-gate-description name="StatefulSetStartOrdinal" >}}
-- {{< feature-gate-description name="StorageVersionAPI" >}}
-- {{< feature-gate-description name="StorageVersionHash" >}}
-- {{< feature-gate-description name="StructuredAuthenticationConfiguration" >}}
-- {{< feature-gate-description name="StructuredAuthorizationConfiguration" >}}
-- {{< feature-gate-description name="TopologyAwareHints" >}}
-- {{< feature-gate-description name="TopologyManager" >}}
-- {{< feature-gate-description name="TopologyManagerPolicyAlphaOptions" >}}
-- {{< feature-gate-description name="TopologyManagerPolicyBetaOptions" >}}
-- {{< feature-gate-description name="TopologyManagerPolicyOptions" >}}
-- {{< feature-gate-description name="TranslateStreamCloseWebsocketRequests" >}}
-- {{< feature-gate-description name="UnauthenticatedHTTP2DOSMitigation" >}}
-- {{< feature-gate-description name="UnknownVersionInteroperabilityProxy" >}}
-- {{< feature-gate-description name="UserNamespacesPodSecurityStandards" >}}
-- {{< feature-gate-description name="UserNamespacesSupport" >}}
-- {{< feature-gate-description name="ValidatingAdmissionPolicy" >}}
-- {{< feature-gate-description name="VolumeAttributesClass" >}}
-- {{< feature-gate-description name="VolumeCapacityPriority" >}}
-- {{< feature-gate-description name="WatchBookmark" >}}
-- {{< feature-gate-description name="WatchList" >}}
-- {{< feature-gate-description name="WindowsHostNetwork" >}}
-- {{< feature-gate-description name="WinDSR" >}}
-- {{< feature-gate-description name="WinOverlay" >}}
-- {{< feature-gate-description name="ZeroLimitedNominalConcurrencyShares" >}}
```

Comment:
This change also reminds me that we may want to revise our [existing guidance](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to align with this change.

> Also make sure to move the relevant list entry and [feature-gate-description shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-gate-description) into the removed feature gates page. The lists are sorted alphabetically.

Reply:
Ah, good point.